src/scenes/UserProfile/UserProfile.jsx

```diff
@@ -90,19 +91,29 @@ class UserProfile extends Component {
             <h1>Identity</h1>
             <form className="card-form">
               <label htmlFor="name">Name</label>
-              <input name="name" className="card-input" value={name} onChange={e => this.onIdentityChange(e)} />
+              <input name="name" className="card-input" value={name} onChange={e => this.onIdentityChange(e)} autoComplete="off" />
               <label htmlFor="email">Email address</label>
               <input name="email" className="card-input" value={email} onChange={e => this.onIdentityChange(e)} />
               <label htmlFor="picture">Picture link</label>
-              <input name="picture" className="card-input" value={picture} onChange={e => this.onIdentityChange(e)} />
-            </form>
+							<div style={{display: 'flex', alignItems: 'center'}}>
+								<Avatar
+                  image={User.picture}
+                  style={{
+                    width: '60px',
+                    height: '60px',
+                    marginRight: '20px'
+                  }}
+                />
```

Comment:
Could you show a screenshot of what it looks like?

Reply:
![image](https://user-images.githubusercontent.com/6669733/38924075-49c4389a-42fc-11e8-842c-99bfe45652ce.png)